### ISSUE TITLE
bug: All options get greyed out if windows explorer pop-out is closed.

### ISSUE THREAD

**Issue by PopTartsWasTaken** [0neGal/viper#77]
**Describe the bug**
If you click "Cancel" or the close button on the windows explorer popout when clicking "Install Mod" in the Northstar "Mods" tab, every option in the Northstar tab will be greyed out until Viper is re-opened, or "Check for updates / Up-to-date" is clicked.
**To Reproduce**
1. Go to the Northstar tab in Viper
2. Click over to the Mods tab
3. Click "Install Mod"
4. Cancel or close out of the explorer pop-out
5. All options will now be greyed out.

**Expected behavior**
Either the buttons should not be greyed out when opening explorer, or the buttons should be clickable again after explorer is closed.

**Screenshots**
(Buttons before opening explorer) https://prnt.sc/26xhfr0
(Buttons while explorer is open) https://prnt.sc/26xhgmf
(Buttons after explorer is closed) https://prnt.sc/26xhh0q

**Version:**
Release 1.2.2

**Additional Info**
Any extra info should go here!

**Comment by 0neGal** [comment]
This should be fixed in the latest commit, and thereby in the next release. For short brevity of why this happened, when a mod starts being installed it disables the buttons to ensure you don't install two mods at the same time leading to problems, and related.

However they originally only get re-enabled once a mod has been installed or failed to install, however since we never selected a mod to install it tries to install "nothing", which leads to an error in the install function therefore never leading to the installed/error popup which is what re-enables the buttons.

We now directly check if a file path was even selected if not we simply tell it to re-enable. This was mostly technical jargon, but I felt like writing it out anyway because I decently bored.


**TL;DR:** The bug is solved and will not exist in the next release...
